### DISCUSSION TITLE
Mapping of numbers

### DISCUSSION THREAD

**Discussion by fabiogaliano** [Anze/KeyCluCask#92] [Q&A]
Hi! First, thank you for this project.

Then, might be dumb but is there any mapping of numbers and equivalent letters/symbols? 

Thank you

**Comment by Anze** [comment]
Hey [fabiogaliano](https://github.com/fabiogaliano),

Thanks for your message! Just to make sure I fully understand could you please give me a bit more detail or an example of what you're thinking about?

**Reply by fabiogaliano** [reply]
Yes, thank you for the help!

```json
{
    "net.kovidgoyal.kitty": {
        "yazi": [
            ["Hotkey 1", "command", 15],
            ["Hotkey 10", "", 39],
            ["Hotkey 11", "", 47],
            ["Hotkey 2", "option", 19],
            ["Hotkey 3", "control", 17],
            ["Hotkey 4", "shift", 14],
            ["Hotkey 5", "", 0],
            ["Hotkey 6", "", 14],
            ["Hotkey 7", "", 34],
            ["Hotkey 8", "", 31],
            ["Hotkey 9", "", 42]
        ]
    }
} 
```
So, these numbers represent a key. I was wondering if you'd have a table with the mapping for each key that could be shared. Like A=0, B=15, etc. I think it would help a lot to share it in the readme also, because we can instantly copy and paste in some llm with documentation of an app and it creates configs super fast.

**Reply by Anze** [reply]
Ah, I see what you mean. While I can say yes, the mapping might not be obvious for keyboards in other languages. 
For example: 
1. Germany
```
Z = 6 for English keyboard (layout)
Y = 16 for English keyboard (layout)

Z = 16 for Germany keyboard (layout)
Y = 6 for Germany keyboard (layout)
```
2. French
```
Q = 12 for English keyboard (layout)
A = 0 for English keyboard (layout)

Q = 0 for French keyboard (layout)
A = 12 for French keyboard (layout)
```

Since letters aren’t strictly tied to specific keys, it could be confusing for users with non-English keyboards. The key number represents the physical position on the keyboard, making it easy to identify the corresponding letter based on the keyboard layout. Unfortunately, the reverse process isn’t as straightforward.

On second thought, mapping it strictly to English letters might simplify things. I’ll keep this in mind for future releases.
